src/Stratis.Bitcoin.IntegrationTests/BlockStore/RetrieveFromBlockStoreSpecification.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Stratis.Bitcoin.IntegrationTests.TestFramework;
+using Xunit;
+
+// Disable warnings about "this" qualifier to make the Specification more readable
+// ReSharper disable ArrangeThisQualifier
+
+namespace Stratis.Bitcoin.IntegrationTests.BlockStore
+{
+    public partial class RetrieveFromBlockStoreSpecification : BddSpecification
+    {
+        [Fact]
+        public void A_block_can_be_retrieved_by_its_uint256_identifier()
```

Comment:
Does the type actually matter in the test name (unit256) ? 

And split the unit test or rename - see my next comment below.